TTT.h

```diff
@@ -0,0 +1,10 @@
+#include "pole.h"
+
+class TTT 
```

Comment:
I think it would be a good idea to avoid non-descriptive object names. For example, `TTTBoard` or `TicTacToe` would be preferred.

It's not that often that one would type out the class name, so I don't think it costs that many extra keystrokes to make a more readable and descriptive name. 😄